dags/scrape_factory.py

```diff
@@ -47,7 +47,7 @@ def get_dag_id(dag_name, dag_config, interval):
         return '{0}_{1}_thru_{2}'.format(dag_name, int_day_map[days[0]], int_day_map[days[-1]])
 
 @provide_session
-def previous_scrape_done(**kwargs):
+def previous_scrape_done(session=None, **kwargs):
     running_scrapes = kwargs['session'].query(TaskInstance).filter(
```

Comment:
Needed to do this, for this to work:

```suggestion
    running_scrapes = session.query(TaskInstance).filter(
```